app/views/content/is-teaching-right-for-me/_facts_and_figures.html.erb

```diff
@@ -0,0 +1,30 @@
+<article>
+  <section class="container">
+      <h2 class="heading--box-blue">Numbers to know</h2>
+
+      <div class="inset">
+        <%= render Content::FactListComponent.new(
+          [
+            {
+              icon: "icon-ticket",
+              value: "£10k+",
```

Comment:
```suggestion
              value: "£15k+",
```
@EllieNodder Can you check this figure? I think it should be 15k - lowest bursary currently?